data/anaconda.conf

```diff
@@ -31,6 +31,15 @@ optional_modules =
     org.fedoraproject.Anaconda.Modules.Subscription
     org.fedoraproject.Anaconda.Addons.*
 
+# Save input kickstart to target system.
+save_input_kickstart = True
```

Comment:
Do you think that the current names of these options are ok? Wouldn't be `copy_input_kickstart` better?